### ISSUE TITLE
[Bug] Feed/Comment  not found

### ISSUE THREAD

**Issue by podcasthosting** [creativetimofficial/ct-nuxt-argon-dashboard-pro#5]
Compilation fails:

```
 ERROR  Failed to compile with 1 errors                                                                                                                                                                                                                                                       friendly-errors 14:57:31

This dependency was not found:                                                                                                                                                                                                                                                                friendly-errors 14:57:31
                                                                                                                                                                                                                                                                                              friendly-errors 14:57:31
* @/components/Feed/Comment in ./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib??vue-loader-options!./components/pages/dashboard/ActivityFeed.vue?vue&type=script&lang=js&                                                                                             friendly-errors 14:57:31
                                                                                                                                                                                                                                                                                              friendly-errors 14:57:31
To install it, you can run: npm install --save @/components/Feed/Comment 
```

The error comes from components/pages/dashboard/ActivityFeed.vue 

`import Comment from '@/components/Feed/Comment';`

It works if you just change it to 

`import Comment from '@/components/feed/Comment';`

so "feed" with a small "f".

**Comment by qcdesign** [comment]
Thanks, this drove me mad for around an hour, even checked to make sure "C"omment wasn't saved as "c"omment.vue

**Comment by cristijora** [comment]
This issue was fixed in version 1.2.0 along with other fixes & improvements.
For more details check the detailed [changelog here](https://demos.creative-tim.com/nuxt-argon-dashboard-pro/documentation/changelog.html)
